build/embedinterop.targets

```diff
@@ -9,42 +9,10 @@
               '%(FileName)' == 'Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.Imaging.Interop.15.0.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.Setup.Configuration.Interop'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Embeddable'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.11.0'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.12.0'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.12.1.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.0.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.5.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.6.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.7.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.8.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.16.0.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.16.1.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.16.2.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.16.3.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.16.4.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.16.5.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.16.6.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.TextManager.Interop.11.0'
-           or '%(FileName)' == 'Microsoft.VisualStudio.TextManager.Interop.12.0'
-           or '%(FileName)' == 'Microsoft.VisualStudio.TextManager.Interop.12.1.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.TextManager.Interop.14.2.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.TextManager.Interop.15.1.DesignTime'
-           or '%(FileName)' == 'Microsoft.VisualStudio.TextManager.Interop.16.0.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.Feedback.Interop.12.0.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.ProjectSystem.Interop'
            or '%(FileName)' == 'microsoft.visualstudio.designer.interfaces'
-           or '%(FileName)' == 'EnvDTE90'
-           or '%(FileName)' == 'EnvDTE100'
            or '%(FileName)' == 'Microsoft.Internal.VisualStudio.Appid.Interop.10.0.DesignTime'
```

Comment:
Should all the assemblies remaining in this `embedinterop.targets` file actually remain? I was under the impression that VS2022 wasn't going to use embed interop at all, in which case we can delete the `embedinterop.targets` file.

Reply:
Removed embedinterop.targets